deployments/scripts/check-containers-version.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright 2019 Globo.com authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+#
+# This script will check the version of all securityTests
+#
+
+rm tmp.txt 2>/dev/null
+
+docker run --rm huskyci/bandit:latest bandit --version > tmp.txt
```

Comment:
@rafaveira3 you don't really need that temp file here.
You could've done it as follows:
`banditVersion=$(docker run --rm huskyci/bandit:latest bandit --version | grep bandit | awk -F " " '{print $2}')`

It seems to work for every container, you just need to change the `grep` and `awk` for each container.

And do you really need this file? You've done everything again at `deployments/scripts/push-containers.sh`

Reply:
Makes sense, @joserenatosilva! Thanks for the tip. I have just done it at cbd8875.